cursorless-talon/src/csv_overrides.py

```diff
@@ -255,37 +256,44 @@ def read_file(
     extra_ignored_values: list[str],
     allow_unknown_values: bool,
 ):
-    with open(path) as f:
-        lines = list(f)
+    with open(path) as csv_file:
+        # Use `skipinitialspace` to allow spaces before quote. `, "a,b"`
+        csv_reader = csv.reader(csv_file, skipinitialspace=True)
+        rows = list(csv_reader)
 
     result = {}
     used_identifiers = []
     has_errors = False
     seen_headers = False
     expected_headers = create_line(*headers)
 
-    for i, raw_line in enumerate(lines):
-        line = raw_line.strip()
-        if not line or line.startswith("#"):
+    for i, row in enumerate(rows):
+        # Remove trailing whitespaces for each cell
+        row = [x.rstrip() for x in row]
+        # Exclude empty or comment rows
+        if len(row) == 0 or (len(row) == 1 and row[0] == "") or row[0].startswith("#"):
             continue
 
         if not seen_headers:
             seen_headers = True
-            if line != expected_headers:
+            if create_line(*row) != expected_headers:
```

Comment:
Let's compare the row directly now instead of round-tripping them through `create_line`?
```suggestion
            if row != headers:
```

Reply:
If you compare to list do you actually compare the content and not that they are the exact same list?

Reply:
Yes.

Reply:
Apparently it does :)